conjur/wrapper/http_wrapper.py

```diff
@@ -55,7 +55,7 @@ def invoke_endpoint(http_verb: HttpVerb, endpoint: ConjurEndpoint, params: dict,
     headers = {}
     if api_token:
         encoded_token = base64.b64encode(api_token.encode()).decode('utf-8')
-        headers['Authorization'] = 'Token token="{}"'.format(encoded_token)
+        headers['Authorization'] = f'Token token="{encoded_token}"'
```

Comment:
is this related to this PR? if not - can you please rearrange the commits to show that?

Reply:
Did it with Matan to fix the broken build